### DISCUSSION TITLE
payment issue

### DISCUSSION THREAD

**Discussion by MESepehr** [airsdk/Adobe-Runtime-Support#1933] [General]
I paid for the basic license of ADOBE AIR SDK on 18 May. My invoice number is AIR400116669. I haven't received your confirmation yet. Would you please check it for me? Thank you

**Comment by ajwfrost** [comment]
Hi

This is showing as 'payment pending' in our database, with the expectation that you'd do a bank transfer (i.e. we've not yet had the notification from the bank for this). Can you send a screenshot or similar for your bank transfer over to the Adobe.Support mailbox at harman.com, and they can chase up the finance team..

thanks

  Andrew

**Comment by MESepehr** [comment]
Hi
I talked to my bank and they said there is no problem from my side. The money has been taken from my account and you need to accept my payment, my bank advised. And please see the attached picture, Screenshot from my bank statement. 
Thank you
![b68259d7-dfdb-480c-9f4e-c14367d4f9c0](https://user-images.githubusercontent.com/4157133/170309887-9c4dd799-27a9-4bd9-a296-2ac027e5fe39.jpg)

**Reply by ajwfrost** [reply]
Okay thanks - I've just heard back from our finance team who have confirmed that they have received this, so we should be able to update the system and get this finalised shortly... apologies for the delay!

thanks
